cacheops/redis.py

```diff
@@ -13,21 +16,28 @@
 logger = logging.getLogger(__name__)
 
 # Global circuit-breaker flag. Values:
-#   None = circuit breaker closed, Redis calls are attempted.
+#   0 = circuit breaker closed, Redis calls are attempted.
 #   <time_seconds> = circuit breaker open, Redis calls are skipped.
-circuit_breaker_opened = None
+try:
+    fd = os.open('/tmp/redis_circuit_breaker', os.O_RDWR)
+    buf = mmap.mmap(fd, 1024)
+except (OSError, ValueError):
+    logger.warn("Mmap's not working, Redis circuit breaker will be local")
+    circuit_breaker = lambda: None  # noqa: E731
+    circuit_breaker.value = 0
+else:
+    circuit_breaker = ctypes.c_int.from_buffer(buf)
```

Comment:
Ideally this would be in a module that we could import here and in our Django core cache code in rover_utils, but I can't get that to work. Django loads the caching modules dynamically and I think it makes imports tricky somehow – plus it doesn't log import errors 😠